src/Pages/Profile/admin/SendUnsubscribeEmail.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import './AdminDashboard.css';
+import Header from '../../../Components/Header/Header';
+import { Button, Container } from 'reactstrap';
+import { getAllUserSubscribedAndVerified } from '../../../APIFunctions/User';
+
+function AdminDashboard(props) {
+  const [buttonText, setButtonText] = useState('Send Unsubscribe Email to All');
+  const [buttonColor, setButtonColor] = useState('');
```

Comment:
nit: could give these states a better name, in case we decide to add more states

something related to the button function would be helpful